scripts/test/devchain.test.ts

```diff
@@ -0,0 +1,75 @@
+import hre from "hardhat";
+import { expect } from "chai";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+
+hre.config.external = {
+  deployments: {
+    hardhat: ["deployments/local"],
+  },
+};
+
+hre.config.networks.hardhat.forking!.blockNumber = 445;
+hre.config.networks.hardhat.accounts = {
+  mnemonic: "concert load couple harbor equip island argue ramp clarify fence smart topic",
+  path: "m/44'/60'/0'/0",
+  initialIndex: 0,
+  count: 20,
+  accountsBalance: "10000000000000000000000",
+};
+
+describe("Deployment check", () => {
+  let multiSig: any;
+  let account: any;
+  let manager: any;
+  let stCELO: any;
+  let rstCELO: any;
```

Comment:
any reason why we have these as any, or no typechain for those 🤔

Reply:
Since this test is checking the proper deployment of staked CELO contracts on the devchain, for use by anyone in a new project. It assumes the staked CELO contract will not be compiled locally and, therefore, no typings generated before running the test.

Reply:
But we're running this in this repo, right, so we do have the typings available?